tests/lib/log.py

```diff
@@ -0,0 +1,23 @@
+import logging
```

Comment:
Reading this after me it is really confusing. I meant just use `logging`, do not use `lib.log`, there is no need to write extra code for this.